model/qti/ImportService.php

```diff
@@ -117,7 +121,26 @@ public function importQTIFile($qtiFile, core_kernel_classes_Class $itemClass, $v
             
             //save itemcontent
             if($qtiService->saveDataItemToRdfItem($qtiItem, $rdfItem)){
+
```

Comment:
It might work but I don't think that this piece of code should be located here. 
Shouldn't this fix be part of oat\taoQtiItem\model\qti\ManifestParserFactory::getResourcesFromManifest() called in  oat\taoQtiItem\model\qti\ImportService::importQTIPACKFile() instead ?